File: app/src/main/java/org/worshipsongs/WorshipSongApplication.java
package org.worshipsongs;

import android.app.Application;
import android.content.Context;

/**
 * Created by Seenivasan on 10/19/2014.
 */
public class WorshipSongApplication extends Application {

    private static Context context;

    public static Context getContext() {
        return context;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        context = getApplicationContext();
    }
}

File: app/src/main/java/org/worshipsongs/WorshipSongApplication.kt
package org.worshipsongs

import android.app.Application
import android.content.Context

/**
 * Created by Seenivasan on 10/19/2014.
 */
class WorshipSongApplication : Application()
{

    override fun onCreate()
    {
        super.onCreate()
        context = applicationContext
    }

    companion object
    {
        var context: Context? = null
    }
}
